User Registration for Shoppers

christian-mack opened the pull request:
Motivation
Shoppers will need the ability to create a profile and login

What You Did
scaffold the components for the login and shopper profile

Trello Card
![Screen Shot 2021-02-02 at 9 21 46 PM](https://user-images.githubusercontent.com/20512418/106702183-c89d7d00-659c-11eb-9aaa-38a42c3337ad.png)

User Story
As a shopper, I want to be able to make/update a profile

Tests
no tested yet. still in development

grownbrand committed:
scaffolding for shopper view

BkAngel201 reviewed (approved):
Nothing to say, all looks good, i like that u used the same folder structure 👍

BkAngel201 commented on .env.sample:
```diff
@@ -1,3 +0,0 @@
-REACT_APP_CLIENT_ID=example
-REACT_APP_OKTA_ISSUER_URI="https://auth.lambdalabs.dev/oauth2/default"
-REACT_APP_API_URI=http://localhost:3000
```
it seems like u need to have this file in the project root folder even when u don't use it

christian-mack replied:
I just removed the `.sample` on that file. I didn't delete it. Not sure if that makes a difference but I can add it back no problem

BkAngel201 replied:
yeah it seems like changing the name of a file will count like delete the one with the old name and create a new one, and that's the conflict

aland1013 replied:
yeah, you have to keep the .env.sample file in the repo because the .env file does not get uploaded, and the GitHub tests will fail without it

christian-mack replied:
I just pushed new changes. added the .env.sample back so we should be good to go now

pjose92 reviewed (dismissed):
Looks good. keep it up

grownbrand committed:
added a route to shopperView

aland1013 reviewed (dismissed):
Looks like a good start. Let me know if you want a hand with any of this.

BkAngel201 reviewed (changes requested):
all looks good, but it seems like .env.sample file need to be in the repo for it to work, apart from, that all looks NICEEE, really good progress team

grownbrand committed:
added routes, form, and button to appBar

grownbrand committed:
user registration form style and functional. need to implement api calls to add user to db.

grownbrand committed:
fixing .env.sample file name change

aland1013 commented:
The .env.sample file is still missing so the tests are failing

grownbrand committed:
removed .env* from .gitignore

aland1013 committed:
Update .env.sample

changed REACT_APP_CLIENT_ID to "example" so we don't have any secrets uploaded to the repo

aland1013 reviewed (dismissed):
Great job, Christian

BkAngel201 reviewed (dismissed):
.env.sample restored Nicely DOne

grownbrand committed:
Merge branch 'main' into add/sellerProfile

pjose92 reviewed (approved):
Looks good. Great job
